app/src/views/Contribution.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+  <template v-if="fullContributions && !loading">
+    <BaseHeader :name="title" />
+    <SectionHeader :title="`Contributions (${contributionTotal})`" />
+    <ContributionDetail :contributions="fullContributions" />
+  </template>
+  <LoadingSpinner v-else />
+</template>
+
+<script lang="ts">
+import BaseHeader from 'src/components/BaseHeader.vue';
+import SectionHeader from 'src/components/SectionHeader.vue';
+import ContributionDetail from 'src/components/ContributionDetail.vue';
+import useDataStore from 'src/store/data';
+import { computed, defineComponent, ref, watch } from 'vue';
+import LoadingSpinner from 'src/components/LoadingSpinner.vue';
+import { filterContributionGrantData } from 'src/utils/data/contributions';
+import useWalletStore from 'src/store/wallet';
+import { useRoute } from 'vue-router';
+import { GrantMetadata, GrantRoundMetadata } from '@dgrants/types';
+
+function setTitle(route: string) {
+  return route === '/contribution/donations' ? 'My Contributions' : 'Contributions';
+}
+
+function contributionDetails() {
+  const { grantContributions, grants, grantMetadata, grantRounds, grantRoundMetadata } = useDataStore();
+  const { userAddress } = useWalletStore();
+
+  const fullContributions = ref();
```

Comment:
This should have a type specified, otherwise it defaults to `any` which we don't want